.clang-format

```diff
@@ -70,14 +80,15 @@ ObjCBlockIndentWidth: 2
 ObjCSpaceAfterProperty: false
 ObjCSpaceBeforeProtocolList: false
 PenaltyBreakBeforeFirstCallParameter: 1
-PenaltyBreakComment: 300
-PenaltyBreakFirstLessLess: 120
-PenaltyBreakString: 1000
-PenaltyExcessCharacter: 1000000
-PenaltyReturnTypeOnItsOwnLine: 200
+PenaltyBreakComment: 100
```

Comment:
Can you explain what you get from adjusting penalties?

Reply:
Sorry didn't see your comments before. I think the number itself is mostly arbitrary so just a matter of how much you want the formatter to avoid a strategy. In my version, I believe the formatter will try splitting a string before it exceeds the column limit.